plugin/pkg/shoot/validator/admission.go

```diff
@@ -279,6 +279,9 @@ func (v *ValidateShoot) Admit(ctx context.Context, a admission.Attributes, o adm
 	if err := validationContext.ensureMachineImages(); err != nil {
 		return err
 	}
+	if err := validationContext.validateShootScaleupToHA(); err != nil {
```

Comment:
Doesn't this check rather belong to static validation, https://github.com/gardener/gardener/blob/42ccdb22065e3f51b0fe4fe343d5af05d6f0ca39/pkg/apis/core/validation/shoot.go#L145-L157 ?

Reply:
Yes

Reply:
Also, unit tests should be added